mindsdb/integrations/handlers/confluence_handler/README.md

```diff
@@ -0,0 +1,52 @@
+# Confluence Handler
+
+Confluence handler for MindsDB provides interfaces to connect with Confluence via APIs and pull confluence data into MindsDB.
+
+## Confluence
+
+Confluence is a collaborative documentation tool,it can be used to host Wiki pages. In this handler,python client of api is used and more information about this python client can be found (here)[https://pypi.org/project/atlassian-python-api/]
+
+
+## Confluence Handler Initialization
+
+The Confluence handler is initialized with the following parameters:
+
+- `url`: Confluence hosted url instance
+- `confluence_api_token`: Confluence API key to use for authentication
```

Comment:
Can we just add a link to the docs for generating the token please?

Reply:
Done. thanks Zoran for reviewing